apps/src/lib/node/ledger/shell/mod.rs

```diff
@@ -129,6 +132,11 @@ pub enum ErrorCodes {
     InvalidOrder = 4,
     ExtraTxs = 5,
     Undecryptable = 6,
+    ReplayTx = 7,
```

Comment:
This is unfortunately not documented in main, but the idea of these numbers is that past a certain threshold, we reject the block. In this branch, those are error codes `>=4`. However, we can't reject an entire block if we get the `InvalidDecryptedChainId` error since we have to include all decrypted txs in order. So its number should be lowered below the critical threshold

Reply:
I think the same is also true of `ExpiredDecryptedTx`